_posts/2019-08-26-how-to-use-gruntwork-infrastructure-as-code-library.adoc

```diff
@@ -238,11 +238,11 @@ IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access thes
   so you know how to run things in dev, how to deploy changes to prod, how to find metrics and logs, how to connect
   over VPN and SSH, and so on. This is a great starting point for exploring the Reference Architecture.
 +
-* https://github.com/gruntwork-io/infrastructure-modules-multi-account-acme[infrastructure-modules]: In this repo,
+* https://github.com/gruntwork-io/terraform-aws-service-catalog[AWS Service Catalog]: In this repo,
   you'll find the reusable modules that define the infrastructure for the entire company (in this case, for Acme).
```

Comment:
This sentence needs rewording, as it is no longer code for a particular company.